README.md

```diff
@@ -0,0 +1,64 @@
+<p align="center">
+  <img src="/images/banner.png">
```

Comment:
maybe as a caption for the image the definitrion of gong i don't know how popular the instrument is